lms/static/scripts/frontend_apps/utils/grader-service.js

```diff
@@ -16,11 +16,10 @@ import { apiCall } from './api';
 /**
  * Submits a student's grade to the LTI endpoint.
  *
- * @param {Student} student - Student object
- * @param {number} grade - A number between 0 and 1
- * @param {string} authToken - The auth token from the config
- * @return {Object} - An empty object
- *
+ * @param {Object} options
+ * @param {Student} options.student - Student object
+ * @param {number} options.grade - A number between 0 and 1
```

Comment:
Curious: is there a way to capture the "between 0 and 1"-ness in TS notation? Or does that wander too far into validation territory?

Reply:
If a number is one of a discrete number of integers you can use `0|1` as the type for example. I don't think you can specify ranges of values for floats or ints though.